libkmod/libkmod-internal.h

```diff
@@ -40,12 +40,8 @@ static _always_inline_ _printf_format_(2, 3) void
 #define KCMD_LINE_SIZE 4096
 
 #ifndef HAVE_SECURE_GETENV
-#  ifdef HAVE___SECURE_GETENV
-#    define secure_getenv __secure_getenv
-#  else
-#    warning neither secure_getenv nor __secure_getenv is available
-#    define secure_getenv getenv
-#  endif
+#warning secure_getenv is not available
+#define secure_getenv getenv
```

Comment:
Intendation should still have two spaces after `#` here.

Reply:
Line 23 etc. have this style. I don't mind either way, just want to highlight consistency as well.

Reply:
I usually prefer not indenting the ifdefs unless it makes it very unreadable.